### ISSUE TITLE
This breaks the new activity icon.

### ISSUE THREAD

**Issue by Peilonrayz** [vyznev/soup#44]
This breaks the new activity style. It should look like:

[![enter image description here][2]][2]

However it looks like this:

[![enter image description here][1]][1]

Removing the following style fixes this for me.

```lang-css
.question-summary .question-hyperlink {
  display: block;
}
```

  [1]: https://i.stack.imgur.com/oOx94.png
  [2]: https://i.stack.imgur.com/HnmdW.png

**Comment by vyznev** [comment]
Huh... I've never seen that icon, and I can't seem to figure out how to make it appear. Any idea how I might be able to reproduce that?

Obviously, I can just disable the problematic fix, and I'll do that if I can't fix it any other way. But I'd prefer to make it work correctly with the icon if I can.

**Comment by Peilonrayz** [comment]
Honestly I don't know how to reproduce it. It seems to appear on my [custom filters](https://meta.stackexchange.com/questions/330326/custom-filters-release-announcement), it may be when I've looked at the list and a question gets edited, but I'm not sure.

If you:

1. Make some custom filters,
2. Save them so they appear on the right hand side of the SE you main at. and
3. Look at the every now and then you should be able to get on to pop up within a day.

Or at least that worked for me.

**Comment by vyznev** [comment]
Thanks, managed to repro it. It looks like adding something like the following CSS should fix it:

```
.question-summary .activity-indicator {
    float: left;
    margin-top: 4px;
}
```

I'll add that to the development branch right now, and consider making a new stable release in a day or two.

**Comment by vyznev** [comment]
Fixed in SOUP v1.56.
